### ISSUE TITLE
usage of (ctrl) shortcuts

### ISSUE THREAD

**Issue by Cre3per** [jackrosenthal/threelayout#50]
Many shortcuts that are left-hand-only on qwerty require the right hand on 3l.  
This makes it inefficient to use 3l in programs where the right hand usually resides on the mouse, e.g. browser, inkscape.

![standard_three (1)](https://github.com/jackrosenthal/threelayout/assets/12541974/95a54f4d-3dac-4363-9397-3a4d601b01cf)

Default shortcuts, for system and surfingkeys, all left-hand-only on qwerty:

- ctrl-c Copy
- ctrl-v Paste
- ctrl-f Find
- x/ctrl-w Close Tab
- ctrl-t New Tab
- r/ctrl-r Refresh
- ctrl-s Save
- Some more niche shortcuts

How do you (Authors/Users) use programs with such shortcuts?  
Why isn't 3l mirrored left-to-right (Seeing that these are common shortcuts)?

**Comment by jackrosenthal** [comment]
I personally find right-handed shortcuts easier.  (E.g., `LCtrl+C` is easy to press as it's on two different hands).

LCtrl+F can be harder to press as it requires a slight hand shift to reach for Ctrl with Lpinky and F with Lring.

However, to each their own!  I've seen variant layouts of other layouts keep QWERTY-style Ctrl combinations --- i'd be open to accept a variant of 3l in this repo for that (i.e., `3l-ctrl-qwerty`).

**Comment by Cre3per** [comment]
@jackrosenthal I agree that right-handed shortcuts are easier to press. The problem arises when the right hand is resting on the mouse and has to be lifted back to the keyboard to press a shortcut.

I'd rather not move individual keys as that would change 3l's typing distance.  
I'm currently experimenting with a full mirror, would something like this be an acceptable variant? (unstable)

![a](https://github.com/jackrosenthal/threelayout/assets/12541974/98d26e32-20ed-47e7-8585-45b4b33a9dea)

![b](https://github.com/jackrosenthal/threelayout/assets/12541974/6c60437b-01e9-4af8-babc-4025c9fc13fe)

(not sure about mirroring all layers)

3l is my first time trying out alternative layouts, all feedback is welcome

**Comment by JellyJoe198** [comment]
I have a similar issue when using the mouse and left hand. Something I'm experimenting with is putting (part of) the qwerty layout on the 2nd layer of my System76 QMK keyboard, so I can just press ctrrl+fn with my pinky and have the qwerty shortcuts. (also Enter because it is used often in CAD)
![Image](https://github.com/user-attachments/assets/1798137a-6995-447f-b50f-bda1d262d18b)
![Image](https://github.com/user-attachments/assets/a5e8a203-2336-451b-9ab0-d91af27bbc6a)
